lib/plugins/features/movies/controller.js

```diff
@@ -1,11 +1,15 @@
 'use strict';
 
-const Movie = require('../../../models/movie');
+const Movie    = require('../../../models/movie');
+const Location = require('../../../models/location');
+
+const LOCATION_MODEL_FETCH_OPTIONS = { require: true };
```

Comment:
While I like the separation of these two variables into class constants to improve modularity, I wonder if it's necessary to have these separated into variables because both variables are JSON objects with one field and each object is only used a max of 3 times. Perhaps getting rid of these variables and including these options directly in the function call would improve code readability/simplicity. You could also do this in other places in this PR for variables that are small JSON objects used only once or twice. What do you think?

Reply:
Fair point but I usually prefer defining constants like this because it 
1. If you need to modify those options, you have one place to do it as opposed to every single place where you've used it. If there were scenarios where these options differed in individual queries, I probably would have defined them in the query itself.
2. It avoids typos :)

Let me know your thoughts on ^^

Reply:
Fair points! Closing this.